src/extensions/spikeamplitudes/SpikeAmplitudesView/SpikeAmplitudesPanel.ts

```diff
@@ -59,15 +59,23 @@ class SpikeAmplitudesPanel {
             }
         }
         else {
-            painter.drawText({xmin: timeRange.min, xmax: timeRange.max, ymin: 0, ymax: 1}, {Horizontal: 'AlignCenter', Vertical: 'AlignCenter'}, font, pen, brush, 'calculating')
+            painter.drawText({
+                rect: {xmin: timeRange.min, xmax: timeRange.max, ymin: 0, ymax: 1},
+                alignment: {Horizontal: 'AlignCenter', Vertical: 'AlignCenter'},
+                font, pen, brush,
+                text: 'calculating'
+            })
         }
     }
+    paintYAxis(painter: CanvasPainter, width: number, height: number) {
```

Comment:
The panel now can optionally paint a Y axis. It gets a painter that is ready to paint in the y axis box (to the left of the plot) and is told the width and heigh in pixels.